--- a/hangouts_media.c
+++ b/hangouts_media.c
@@ -138,7 +138,7 @@
 		}
 	} else if (purple_session_type & PURPLE_MEDIA_VIDEO) {
 		return MEDIA_TYPE__MEDIA_TYPE_VIDEO;
-#if PURPLE_VERSION_CHECK(2, 10, 12) || PURPLE_VERSION_CHECK(3, 0, 0)
+#if PURPLE_VERSION_CHECK(2, 10, 11) || PURPLE_VERSION_CHECK(3, 0, 0)
 	} else if (purple_session_type & PURPLE_MEDIA_APPLICATION) {
 		return MEDIA_TYPE__MEDIA_TYPE_DATA;
 #endif
@@ -209,7 +209,7 @@
 			case PURPLE_MEDIA_NETWORK_PROTOCOL_UDP:
 				ice_candidate->protocol = PROTOCOL__UDP;
 				break;
-#if PURPLE_VERSION_CHECK(2, 10, 12) || PURPLE_VERSION_CHECK(3, 0, 0)
+#if PURPLE_VERSION_CHECK(2, 10, 11) || PURPLE_VERSION_CHECK(3, 0, 0)
 			case PURPLE_MEDIA_NETWORK_PROTOCOL_TCP_PASSIVE:
 			case PURPLE_MEDIA_NETWORK_PROTOCOL_TCP_ACTIVE:
 			case PURPLE_MEDIA_NETWORK_PROTOCOL_TCP_SO:
